src/vs/workbench/services/positronConnections/browser/positronConnectionsInstance.ts

```diff
@@ -461,7 +467,7 @@ class PositronConnectionItem implements IPositronConnectionItem {
 
 	private async getIcon() {
 		const icon = await this.instance.client.getIcon(this.path);
-		if (icon === '') {
+		if (!icon || icon === '') {
```

Comment:
```suggestion
		if (!icon) {
```

Super nit: `''` is falsey, so this can be simplified.